internal/cmd/pods/pods.go

```diff
@@ -0,0 +1,208 @@
+	for _, component := range pod.Component {
+		if len(component.Name) == 0 {
+			continue
+		}
+
+		nameComponents := []string{component.Name}
+		if len(component.Version) > 0 {
+			nameComponents = append(nameComponents, component.Version)
+		}
+		components = append(components, strings.Join(nameComponents, ":"))
+	}
+
+	if len(components) == 0 {
+		components = append(components, "-")
+	}
+
+	caser := cases.Title(language.English)
+	return []string{
+		caser.String(pod.Status),
+		pod.Name,
+		strings.Join(components, ", "),
+		fmt.Sprintf("%d/%d", readyContainers, totalContainers),
+		caser.String(pod.Phase),
+		fmt.Sprint(totalRestart),
+		HumanDuration(time.Since(pod.StartTime)),
+	}
+}
+
+// adapted from https://github.com/kubernetes/apimachinery/blob/master/pkg/util/duration/duration.go#L48
+// HumanDuration returns a succinct representation of the provided duration with limited precision for
+// consumption by humans. It provides ~2-3 significant figures of duration.
+func HumanDuration(d time.Duration) string {
+	var durationString string
+
+	switch {
+	case d < time.Minute*2: // show seconds duration until 2 minutes
+		convertedDuration := d.Round(time.Second) / time.Second
+		durationString = fmt.Sprintf("%ds", convertedDuration)
+	case d < time.Minute*10: // show minutes and seconds duration until 10 minutes
+		convertedDuration := d.Round(time.Second)
+		durationString = fmt.Sprint(convertedDuration)
+	case d < time.Hour*3: // show minutes duration until 3 hours
+		convertedDuration := d.Round(time.Second) / time.Minute
+		durationString = fmt.Sprintf("%dm", convertedDuration)
+	case d < time.Hour*8: // show hours and minutes duration until 8 hours
+		convertedDuration := d.Round(time.Minute) / time.Minute
+		durationString = fmt.Sprintf("%dh%dm", convertedDuration/60, convertedDuration%60)
+	case d < time.Hour*48: // show hours duration until 2 days
+		convertedDuration := d.Round(time.Minute) / time.Hour
+		durationString = fmt.Sprintf("%dh", convertedDuration)
+	case d < time.Hour*192: // show days and hours duration until ~ 8 days (24 h * 8 days = 192 hours)
+		convertedDuration := d.Round(time.Minute) / time.Hour
+		residualHours := convertedDuration % 24
+		if residualHours == 0 {
+			durationString = fmt.Sprintf("%dd", convertedDuration/24)
+		} else {
+			durationString = fmt.Sprintf("%dd%dh", convertedDuration/24, residualHours)
+		}
+	case d < time.Hour*8760: // show days duration until ~ 1 year (24 h * 365 days = 8760 hours)
+		convertedDuration := d.Round(time.Hour) / time.Hour
+		durationString = fmt.Sprintf("%dd", convertedDuration/24)
+	default: // show days and years duration after the first year
+		convertedDuration := (d.Round(time.Hour) / time.Hour) / 24
+		residualDays := convertedDuration % 365
+		if residualDays == 0 {
+			durationString = fmt.Sprintf("%dy", convertedDuration/365)
+		} else {
+			durationString = fmt.Sprintf("%dy%dd", convertedDuration/365, residualDays)
+		}
+	}
+
+	return durationString
```

Comment:
Might it be better to move this function into an util? 
Is this something that might be useful in other cases?
(e.g. jobs ? )

Reply:
You're right, I'm moving it out